its/ruling/src/test/resources/expected/php-S4144.json

```diff
@@ -0,0 +1,154 @@
+{
+'project:PHPExcel_1.7.9_doc/Classes/PHPExcel/Chart/Layout.php':[
+198,
+207,
+],
+'project:PHPExcel_1.7.9_doc/Classes/PHPExcel/Reader/Excel5/Escher.php':[
+396,
+426,
+486,
+498,
+515,
+586,
+],
+'project:PHPExcel_1.7.9_doc/Classes/PHPExcel/Shared/trend/bestFitClass.php':[
+143,
+164,
+],
+'project:symfony/Symfony/Bridge/Propel1/Tests/Fixtures/Item.php':[
+74,
+79,
+92,
+],
+'project:symfony/Symfony/Bridge/Propel1/Tests/Fixtures/TranslatableItem.php':[
+67,
+72,
+85,
+],
+'project:symfony/Symfony/Bridge/Propel1/Tests/Fixtures/TranslatableItemI18n.php':[
+60,
+65,
+78,
+],
+'project:symfony/Symfony/Component/CssSelector/XPath/Extension/AbstractExtension.php':[
+35,
+43,
+51,
+59,
+],
+'project:symfony/Symfony/Component/DependencyInjection/LazyProxy/PhpDumper/NullDumper.php':[
+42,
+],
+'project:symfony/Symfony/Component/DependencyInjection/Tests/Dumper/XmlDumperTest.php':[
+43,
+],
+'project:symfony/Symfony/Component/Form/Button.php':[
+222,
+232,
+242,
+282,
+332,
+342,
+],
+'project:symfony/Symfony/Component/Form/ButtonBuilder.php':[
+370,
```

Comment:
Shouldn't we exclude cases where there's only one statement which is a throw?

Reply:
I don't think so : those are accessors so I tend to think the implementation should be factorized in that peculiar case.